docs/crds/clustercompliance-report.md

```diff
@@ -0,0 +1,744 @@
+# ClusterComplianceReport
+
+The ClusterComplianceReport is a cluster-scoped resource, which represents the latest compliance control checks results.
+The report spec defines a mapping between pre-defined compliance control check ids to security scanners check ids.
+Currently, only `kube-bench` and `config-audit` security scanners are supported
+
+
+The NSA compliance report is composed of two parts :
+
+- `spec:` represents the compliance control checks specification, check details, and the mapping to the security scanner (this part is defined by the user)
+
+- `status:` represents the compliance control checks (as defined by spec mapping) results extracted from the security scanners reports (this part is output by starboard)
+
+The following shows a sample ClusterComplianceReport NSA specification associated with the `cluster`
+
+```json
+{
+  "apiVersion": "aquasecurity.github.io/v1alpha1",
+  "kind": "ClusterComplianceReport",
+  "metadata": {
+    "annotations": {
+      "kubectl.kubernetes.io/last-applied-configuration": ""
+    },x§
```

Comment:
Some type here? Can you also confirm that you want to use JSON in examples even though we use YAML in other places?